docs/other-api/serve.md

```diff
@@ -37,6 +37,25 @@ In development, `remix-serve` will ensure the latest code is run by purging the
   }
   ```
 
+  If you need a workaround for preserving cache in development, you can store it in the global variable.
+
+  ```ts lines=[1-9] 
+  // since the cache is stored in global it will only 
+  // be recreated when you restart your dev server.
+  const cache = (() => {
+    if (!global.uniqueCacheName) {
+      global.uniqueCacheName = new Map()
+    }
+
+    return global.uniqueCacheName
+  })
+
+
+  export async function loader({ params }) {
+    // ...
```

Comment:
I think it would be good if we added few more lines of code in the `loader` just like in the example from like 29-37 in this file 😄 

At a first glance, it may not be clear to people if the need to do use the cache like this:

```
export async function loader({ params }) {
    if (global.uniqueCacheName.has(params.foo)) {
```
or

```
export async function loader({ params }) {
   if (cache.has(params.foo)) {
```
What do you think?

Reply:
Yeah I see how that can be confusing. I've added some more lines of example code to the loader function.